modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -104,40 +103,31 @@
 	alt_titles = list("Nanotrasen Representative", "Nanotrasen Diplomat", "Central Command Representative")
 
 /datum/job/orderly
-
+	alt_titles = list("Orderly", "Medical Guard") //goof requested that alt dept guards titles be kept to [department] guard
 
 /datum/job/paramedic
 	alt_titles = list("Paramedic", "Emergency Medical Technician", "Search and Rescue Technician")
 
 /datum/job/prisoner
-	alt_titles = list("Prisoner", "Minimum Security Prisoner", "Maximum Security Prisoner", "SuperMax Security Prisoner", "Protective Custody Prisoner")
+	alt_titles = list("Prisoner", "Minimum Security Prisoner", "Maximum Security Prisoner", "SuperMax Security Prisoner", "Protective Custody Prisoner", "Convict", "Felon", "Inmate")
 
 /datum/job/psychologist
 	alt_titles = list("Psychologist", "Psychiatrist", "Therapist", "Counsellor")
 
 /datum/job/quartermaster
-	alt_titles = list("Quartermaster", "Deck Chief", "Cargo Foreman")
+	alt_titles = list("Quartermaster", "Deck Chief", "Cargo Foreman", "Head of Cargo", "Chief Requisition Officer")
 
 /datum/job/research_director
 	alt_titles = list("Research Director", "Silicon Administrator", "Lead Researcher", "Biorobotics Director", "Research Supervisor", "Chief Science Officer")
 
 /datum/job/roboticist
 	alt_titles = list("Roboticist", "Biomechanical Engineer", "Mechatronic Engineer")
 
-/datum/job/science_guard
+/datum/job/science_guard //none for them, goof request :(
 
 
 /datum/job/scientist
-	alt_titles = list(
-    "Scientist",
-    "Circuitry Designer",
-    "Xenobiologist",
-    "Cytologist",
-    "Plasma Researcher",
-    "Anomalist",
-    "Lab Technician",
-    "Theoretical Physicist",
-	)
+	alt_titles = list("Scientist", "Circuitry Designer","Xenobiologist", "Cytologist", "Plasma Researcher", "Anomalist", "Lab Technician", "Theoretical Physicist", "Ordnance Technician")
```

Comment:
The list was done this way to improve readability.
```suggestion
	alt_titles = list(
		"Scientist",
		"Circuitry Designer",
		"Xenobiologist",
		"Cytologist",
		"Plasma Researcher",
		"Anomalist",
		"Lab Technician",
		"Theoretical Physicist",
		"Ordnance Technician"
	)
```

Reply:
i changed it because it was inconsistent with the rest